EventView-1/src/main/java/com/eventview/controller/UsersRestController.java

```diff
@@ -27,74 +26,59 @@
         List<Users> users = userService.getAllUsers();
         if (users == null || users.isEmpty()) {
             log.info("no users found");
-            return new ResponseEntity<List<Users>>(HttpStatus.NO_CONTENT);
+            return new ResponseEntity<>(HttpStatus.NO_CONTENT);
         }
         log.info("users received");
-        return new ResponseEntity<List<Users>>(users, HttpStatus.OK);
+        return new ResponseEntity<>(users, HttpStatus.OK);
     }
 
-    @GetMapping(path = "/user/{userid}")
-    public ResponseEntity<Users> findByUserId(@PathVariable("userid") Integer userid) {
-        log.info("getting user by userid{}", userid);
-        Users users = userService.findByUserId(userid);
+    @GetMapping(path = "/user/{userId}")
+    public ResponseEntity<Users> findByUserId(@PathVariable("userId") Integer userId) {
+        log.info("getting user by userId{}", userId);
+        Users users = userService.findByUserId(userId);
 
-        if (users == null) {
-            log.info("user with the id{} doesn't exist",userid);
-            return new ResponseEntity<Users>(HttpStatus.NOT_FOUND);
-        }
-
-        return new ResponseEntity<Users>(users, HttpStatus.OK);
+        return new ResponseEntity<>(users, HttpStatus.OK);
     }
 
     @PostMapping(path = "/user")
     public ResponseEntity<Void>
     createUser(@RequestBody Users user) {
         log.info("creating new user:{}", user);
```

Comment:
no payload validations for all requests , what if i sent without firstname, it throws error. u shd have payload validations.

Reply:
yeah understood i'll make validators for them